migration/src/test/resources/postgres/init.sql

```diff
@@ -0,0 +1,25 @@
+DROP TABLE IF EXISTS public.apjdbc_schema_history;
+
+DROP TABLE IF EXISTS public.journal;
+
+CREATE TABLE IF NOT EXISTS public.journal (
+  ordering BIGSERIAL,
+  persistence_id VARCHAR(255) NOT NULL,
+  sequence_number BIGINT NOT NULL,
+  deleted BOOLEAN DEFAULT FALSE NOT NULL,
+  tags VARCHAR(255) DEFAULT NULL,
+  message BYTEA NOT NULL,
+  PRIMARY KEY(persistence_id, sequence_number)
+);
+
+CREATE UNIQUE INDEX journal_ordering_idx ON public.journal(ordering);
+
+DROP TABLE IF EXISTS public.snapshot;
+
+CREATE TABLE IF NOT EXISTS public.snapshot (
+  persistence_id VARCHAR(255) NOT NULL,
+  sequence_number BIGINT NOT NULL,
+  created BIGINT NOT NULL,
+  snapshot BYTEA NOT NULL,
+  PRIMARY KEY(persistence_id, sequence_number)
+);
```

Comment:
I would add this file as the V1 file of a migration. We will need a different set of files for each supported DB. 

Then we can create the table as it is now and apply V2 (creating a manifest column and unwrapping the payload), apply V3 to migrate the tags to a separated table, etc.